projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py

```diff
@@ -18,29 +18,64 @@
 
 class MetaPluginConfiguration:
     def __init__(self, config: Configuration):
+        """
+
+        :param config: the DAG plugin configuration
+        """
         self.__config = config
 
     def meta_jobs_delayed_jobs_min_delay_seconds(self):
+        """
+        Returns the minimum delay time for a delayed job to be executed in seconds.
+
+        :return: the number of seconds for the minimum delay of a delayed job
+        """
         return self.__config.get_value(META_JOBS_DELAYED_JOBS_MIN_DELAY_SECONDS)
 
     def meta_jobs_delayed_jobs_randomized_added_delay_seconds(self):
+        """
+        Returns the additional randomized delay time in seconds to the minimum delay time of a delayed job.
```

Comment:
Refer to the detailed configuration description. You can use seealso in reStructuredText I think: 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso

Reply:
Done although I'm not sure this is the best way to do these directives.